sourcegraph/sourcegraph
# Replace DevToolTime with Discord in search footer

Signed-off-by: Justin Dorfman <justin.dorfman@sourcegraph.com>

Replaces the DevToolTime logo/ad with a Discord server invite.

[Slack discussion](https://sourcegraph.slack.com/archives/C015P61EXV1/p1660321223979559)

## Test plan

- Is the svg logo optimized
- Make sure link works
- Grammar check
- Run by Fabiana (branding)



## App preview:

- [Web](https://sg-web-jd-update-search-footer.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kucszqcvsv.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
